stripe-ui-core/detekt-baseline.xml

```diff
@@ -44,5 +44,6 @@
     <ID>ThrowingExceptionsWithoutMessageOrCause:TransformAddressToElement.kt$IllegalArgumentException()</ID>
     <ID>ThrowsCount:UiUtils.kt$@SuppressLint("DiscouragedApi") internal fun Context.getResourceId(uri: Uri): Pair&lt;Resources, Int></ID>
     <ID>TooManyFunctions:StripeTheme.kt$com.stripe.android.uicore.StripeTheme.kt</ID>
+    <ID>UnusedPrivateMember:PhoneNumberElementUI.kt$focusRequester: FocusRequester = remember { FocusRequester() }</ID>
```

Comment:
What up with this?

Reply:
Oh, that seems wrong. Will look into it!